### ISSUE TITLE
rCharts: Error in UseMethod("rPlot") :

### ISSUE THREAD

**Issue by danishmohdkhan** [ramnathv/rCharts#741]
Please help in resolving the issue  : 
 #Here is the code
   require(devtools)
   install_github("ramnathv/rCharts")
   library(rCharts)
   rPlot(mpg ~ wt | am + vs, data = mtcars, type = "point", color = "gear")
 
 #Here is the error
  Error in UseMethod("rPlot") : 
  no applicable method for 'rPlot' applied to an object of class "formula"

**Comment by louise-rb-dupuis** [comment]
Hi,
It is also the error I get in my issue #740 that I opened two weeks ago :) It looks like it happens because of the new version of R. What version are you working with ?

Br,
Louise

**Comment by danishmohdkhan** [comment]
> Hi,
> It is also the error I get in my issue #740 that I opened two weeks ago :) It looks like it happens because of the new version of R. What version are you working with ?
> 
> Br,
> Louise

Yes I recently updated my version of R to 4.0. So rCharts not working in the newer version. Now i have uninstalled 4.0 version and installed again the older 3.6.3 version. rCharts working fine now in the older version.
